ui/assets/css/hound.css

```diff
@@ -252,6 +255,8 @@ button:focus {
   display: block;
   line-height: 30px;
   background-color: #f5f5f5;
+  position: sticky;
+  top: 37px;
```

Comment:
I'd propose changing this to:

```
top: 5%;
```

in order to accommodate the changes with padding for repo title.